packages/beacon-node/src/api/impl/beacon/blocks/index.ts

```diff
@@ -186,17 +189,24 @@ export function getBeaconBlockApi({
 
       metrics?.registerBeaconBlock(OpSource.api, seenTimestampSec, signedBlock.message);
 
-      await Promise.all([
+      // TODO EIP-4844: Open question if broadcast to both block topic + block_and_blobs topic
+      const blockForImport: BlockImport =
+        config.getForkSeq(signedBlock.message.slot) >= ForkSeq.eip4844
+          ? {block: signedBlock, blobs: chain.getBlobsSidecar(signedBlock.message as eip4844.BeaconBlock)}
+          : {block: signedBlock, blobs: null};
+
+      await promiseAllMaybeAsync([
         // Send the block, regardless of whether or not it is valid. The API
         // specification is very clear that this is the desired behaviour.
-        network.gossip.publishBeaconBlock(signedBlock),
-
-        chain.processBlock(signedBlock).catch((e) => {
-          if (e instanceof BlockError && e.type.code === BlockErrorCode.PARENT_UNKNOWN) {
-            network.events.emit(NetworkEvent.unknownBlockParent, signedBlock, network.peerId.toString());
-          }
-          throw e;
-        }),
+        () => network.publishBeaconBlockMaybeBlobs(blockForImport),
+
+        () =>
+          chain.processBlock(blockForImport).catch((e) => {
+            if (e instanceof BlockError && e.type.code === BlockErrorCode.PARENT_UNKNOWN) {
```

Comment:
what scenario this would be possible, PBS?

Reply:
This is an API, an external actor can submit a block for unknown parent. In fact this is exactly what we do in the unknown root sync test from @nazarhussain